src/main/kotlin/de/novatec/betting/game/teams/model/Teams.kt

```diff
@@ -0,0 +1,10 @@
+package de.novatec.betting.game.teams.model
+
+data class Teams(val teams: List<Team>)
```

Comment:
Die Data Class finde ich fast unnötig. Da würde ich tatsählich an den Stellen einfach `List<Team> `verwenden.
Also bspw.
`fun getTeams(season: String): List<Team>`

Reply:
Hab ich mir auch überlegt, tatsächlich würde ich das gerne so lassen (zumindest für den Anfang).  `Teams`, also eine 'Liste' von `Team`s , ist unsere Schnittstelle - da sind wir flexibler, wenn wir ein separates entity verwenden (-> everything is a class)